### PR TITLE
feat: mobile UI

### PR DESCRIPTION
**Pull request by nathaliabj**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: 

### Description:
This PR adds a friendly mobile UI to inform the user that Lightdash does not render on small screens, and instructs the user to use Lightdash on desktop.

### Preview:
Screenshot 2022-01-07 at 09 05 25

### Scope of the changes (select all that apply):
- [x] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: packages/frontend/src/pages/Welcome.tsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ const Welcome: FC = () => {
                         Lightdash Documentation
                     </AnchorButton>
                     <AnchorButton
-                        href="https://community.lightdash.com/"
+                        href="https://github.com/lightdash/lightdash/discussions"
```
Comment (nathaliabj) [inline packages/frontend/src/pages/Welcome.tsx:182]: @ZeRego @owlas I have updated this link to go to our discussions space

**Review by ZeRego — COMMENTED** [review]
- I would rather have this logic in a single place ( eg App.tsx ) and above the route logic since the mobile view should display independently of the route.
- Not sure if we should rely on screen size to determine the mobile view since it won't include tablets. Would rather check the browser agent. While on desktops we can enforce a minimum body width of 768px.

File: packages/frontend/src/hooks/useBreakpoint/index.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+        // on unmount of component remove event listener
+        return () => window.removeEventListener('resize', handleResize);
+    }, [width]);
+
+    return {
+        isOverBreakpoint,
+    };
+};
+
```
Comment (nathaliabj) [inline packages/frontend/src/hooks/useBreakpoint/index.ts:32]: @ZeRego I left the hook here as i feel it could be useful in the future

**Review by ZeRego — APPROVED** [review]
this looks good to me :) We just need to check why the preview environment doesn't work properly.

**Comment by ZeRego** [comment]
@all-contributors add @nathaliabj for code

**Pull request merged**